### PR TITLE
Adding some description

### PR DESCRIPTION
**Pull request by hidogo**
Adding description of

### CODE REVIEW

**Comment by hidogo** [comment]
Adding some description

File: doc/docs.md [hunk 1]
```diff
@@ -914,14 +914,19 @@ There is no explicit declaration of intent, no "implements" keyword.
...
+    green
+    blue
 }
 
 var color := Color.red
 // V knows that `color` is a `Color`. No need to use `color = Color.green` here.
 color = .green
 println(color) // "1"  TODO: print "green"?
 ```
+An enumeration, also referred to as enum is a simultaneous definition of a nominal enumerated type as well as a set of constructors, that can be used to create or pattern-match values of the corresponding enumerated type.enables you to work with those values in a type-safe way within your code.
```
Comment (JalonSolov) [inline doc/docs.md:927]: Needs a comma after 'type', as in "enumerated type, as well as"

File: doc/docs.md [hunk 2]
```diff
@@ -1025,6 +1030,8 @@ posts_repo := new_repo<Post>(db)
 user := users_repo.find_by_id(1)?
 post := posts_repo.find_by_id(1)?
 ```
+Generics enables you to write flexible, reusable functions and types that can work with any type.
+generics method declarations have a type parameter section delimited by angle brackets `<T>` that precedes the method's return type.
```
Comment (JalonSolov) [inline doc/docs.md:1034]: "generics" should be "Generic".  Capitalize at the beginning of a sentence, and singular to match the sentence intent.

File: doc/docs.md [hunk 3]
```diff
@@ -1025,6 +1030,8 @@ posts_repo := new_repo<Post>(db)
 user := users_repo.find_by_id(1)?
 post := posts_repo.find_by_id(1)?
 ```
+Generics enables you to write flexible, reusable functions and types that can work with any type.
```
Comment (JalonSolov) [inline doc/docs.md:1033]: "enables" should be "enable".  "types that can work with any type." is awkward/redundant phrasing.  Perhaps something like "placeholders for any type."

**Comment by hidogo** [comment]
fix typo.

File: doc/docs.md [hunk 4]
```diff
@@ -914,14 +914,19 @@ There is no explicit declaration of intent, no "implements" keyword.
...
+    green
+    blue
 }
 
 var color := Color.red
 // V knows that `color` is a `Color`. No need to use `color = Color.green` here.
 color = .green
 println(color) // "1"  TODO: print "green"?
 ```
+An enumeration, also referred to as enum is a simultaneous definition of a nominal enumerated type,as well as a set of constructors, that can be used to create or pattern-match values of the corresponding enumerated type.enables you to work with those values in a type-safe way within your code.
```
Comment (spytheman) [inline doc/docs.md:927]: What does 'nominal enumerated type' mean?
Also, what do you mean by a 'set of constructors' ?
Copying from https://doc.rust-lang.org/reference/items/enumerations.html is not nice. V's enums are not Rust's enums.

Comment (spytheman) [reply doc/docs.md:927]: V's enums are more like C's enums - they are just a set of numbers/integers.

**Comment by medvednikov** [comment]
Let's not copy paste docs from Rust docs, especially when documenting things that don't even exist in V.

**Pull request closed without merging**